usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -233,7 +234,7 @@ def _handle_string_cast(val: str) -> Union[str, dict]:
     """
     try:
         return json.loads(val)
-    except TypeError:
+    except (TypeError, json.decoder.JSONDecodeError):
```

Comment:
We definitely wan to swallow this error? If so you should put a code comment in explaining why this is ok.

Reply:
Thought I needed it until solving the issue of the delta <-> postgres column mismatch. Fixed